backend/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 .env
+.env.example
```

Comment:
My reasoning was that if people want to rename the file, instead of creating a new `.env` file, then git won't care. 
However this depends on whether or not we anticipate on modifying this example file. If we do, then it shouldn't be added to the .gitignore.

Any other reason why it shouldn't be added? (I might be missing something obvious lol)

Reply:
Sweet, totally agree with you. I totally missed the line in the file that mentions the user should create a new copy - My bad. I'll revert the change now!